qiime2/sdk/result.py

```diff
@@ -262,7 +264,7 @@ def _from_view(cls, type, view, view_type, provenance_capture):
         recorder = provenance_capture.transformation_recorder('return')
         transformation = from_type.make_transformation(to_type,
                                                        recorder=recorder)
-        result = transformation(view)
+        result = transformation(view, level)
```

Comment:
Maybe `validate_level` here as well, but in the `validate` methods, `level` on its own is perfectly fine.